src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/conversation/ConversationEntityAssert.kt

```diff
@@ -0,0 +1,134 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.conversation
+
+import org.assertj.core.api.AbstractObjectAssert
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.INTERNALLY_MANAGED_FIELDS
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa.entity.actionplan.GoalEntity
+import java.time.Instant
+import java.util.UUID
+import java.util.function.Consumer
+
+fun assertThat(actual: ConversationEntity?) = ConversationEntityAssert(actual)
+
+class ConversationEntityAssert(actual: ConversationEntity?) : AbstractObjectAssert<ConversationEntityAssert, ConversationEntity?>(actual, ConversationEntityAssert::class.java) {
```

Comment:
I love love love assertJ and the fact we can write our own custom assertion classes 👍